bin/heroics-generate

```diff
@@ -26,15 +30,32 @@ option_parser = OptionParser.new do |opts|
 end
 
 option_parser.parse!
-if ARGV.length != 3
-  puts option_parser
-else
-  module_name, schema_filepath, url = ARGV
-  schema = Heroics::Schema.new(MultiJson.decode(open(schema_filepath).read))
-  cache = 'Moneta.new(:Memory)'
-  if options[:cache_path]
-    cache = "Moneta.new(:File, dir: \"#{options[:cache_path]}\")"
+
+case ARGV.length
+when 1
+  config_file = ARGV[0]
+  require config_file
+
+  puts Heroics.generate_client
+when 3
```

Comment:
Comments in cb596eeed6bcefeecbe41842a364928ec4396906. Not sure about extraction at this point, but it'd be nice if `ARGV` was less of an opaque thing that we pull magical values out of.